.config/zepter.yaml

```diff
@@ -0,0 +1,39 @@
+version:
+  format: 1
+  # Minimum zepter version that is expected to work. This is just for printing a nice error
+  # message when someone tries to use an older version.
+  binary: 0.13.2
+
+# The examples in the following comments assume crate `A` to have a dependency on crate `B`.
+workflows:
+  check:
+    - [
+        "lint",
+        # Check that `A` activates the features of `B`.
+        "propagate-feature",
+        # These are the features to check:
+        "--features=arbitrary,std,serde,test-utils,metrics",
+        # Do not try to add a new section into `[features]` of `A` only because `B` exposes that feature. There are edge-cases where this is still needed, but we can add them manually.
+        "--left-side-feature-missing=ignore",
+        # Ignore the case that `A` is outside of the workspace. Otherwise it will report errors in external dependencies that we have no influence on.
+        "--left-side-outside-workspace=ignore",
+        # Only check normal dependencies.
+        # Propagating to dev-dependencies leads to compilation issues: https://github.com/alloy-rs/alloy/pull/2144#issuecomment-2702291349
+        "--dep-kinds=normal:check,dev:ignore",
```

Comment:
Can we build this set of checks ourselves instead of cp from alloy

Reply:
Yeah for sure. Drafted for now as I experiment